control-plane-operator/controllers/hostedcontrolplane/ingressoperator/ingressoperator.go

```diff
@@ -0,0 +1,115 @@
+	KubeRBACProxyImage      string
+	ReleaseVersion          string
+	TokenMinterImage        string
+	AvailabilityProberImage string
+}
+
+func NewParams(version string, images map[string]string) Params {
+	return Params{
+		IngressOperatorImage:    images["cluster-ingress-operator"],
+		HAProxyRouterImage:      images["haproxy-router"],
+		ReleaseVersion:          version,
+		TokenMinterImage:        images["token-minter"],
+		AvailabilityProberImage: images[util.AvailabilityProberImageName],
+	}
+}
+
+func ReconcileDeployment(dep *appsv1.Deployment, params Params, apiPort *int32) {
+	dep.Spec.Replicas = utilpointer.Int32(1)
+	dep.Spec.Selector = &metav1.LabelSelector{MatchLabels: map[string]string{"name": "ingress-operator"}}
+	dep.Spec.Strategy.Type = appsv1.RecreateDeploymentStrategyType
+	if dep.Spec.Template.Annotations == nil {
+		dep.Spec.Template.Annotations = map[string]string{}
+	}
+	dep.Spec.Template.Annotations["target.workload.openshift.io/management"] = `{"effect": "PreferredDuringScheduling"}`
+	if dep.Spec.Template.Labels == nil {
+		dep.Spec.Template.Labels = map[string]string{}
+	}
+	dep.Spec.Template.Labels["name"] = "ingress-operator"
+	dep.Spec.Template.Spec.AutomountServiceAccountToken = utilpointer.BoolPtr(false)
+	dep.Spec.Template.Spec.Containers = []corev1.Container{
+		{
+			Command: []string{
+				"ingress-operator",
+				"start",
+				"--namespace",
+				"openshift-ingress-operator",
+				"--image",
+				"$(IMAGE)",
+				"--canary-image",
+				"$(CANARY_IMAGE)",
+				"--release-version",
+				"$(RELEASE_VERSION)",
+			},
+			Env: []corev1.EnvVar{
+				{Name: "RELEASE_VERSION", Value: params.ReleaseVersion},
+				{Name: "IMAGE", Value: params.HAProxyRouterImage},
+				{Name: "CANARY_IMAGE", Value: params.IngressOperatorImage},
+				{Name: "KUBECONFIG", Value: "/etc/kubernetes/kubeconfig"},
+			},
+			Name:            "ingress-operator",
+			Image:           params.IngressOperatorImage,
+			ImagePullPolicy: corev1.PullIfNotPresent,
+			Resources: corev1.ResourceRequirements{Requests: corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("10m"),
+				corev1.ResourceMemory: resource.MustParse("56Mi"),
+			}},
+			TerminationMessagePolicy: corev1.TerminationMessageFallbackToLogsOnError,
+			VolumeMounts: []corev1.VolumeMount{
+				{Name: "ingress-operator-kubeconfig", MountPath: "/etc/kubernetes"},
+				{Name: "serviceaccount-token", MountPath: "var/run/secrets/openshift/serviceaccount"},
+			},
+		},
+		{
+			Name:    "token-minter",
+			Command: []string{"/usr/bin/token-minter"},
+			Args: []string{
+				"-service-account-namespace=openshift-ingress-operator",
+				"-service-account-name=ingress-operator",
+				"-token-file=/var/run/secrets/openshift/serviceaccount/token",
+				"-kubeconfig=/etc/kubernetes/kubeconfig",
+			},
+			Image: params.TokenMinterImage,
+			VolumeMounts: []corev1.VolumeMount{
+				{Name: "serviceaccount-token", MountPath: "var/run/secrets/openshift/serviceaccount"},
```

Comment:
`var/run/secrets/openshift/serviceaccount` is missing a `/` prefix... does it happen to work as-is by coincidence?